### ISSUE TITLE
No generated values for "out of frame" attribute

### ISSUE THREAD

**Issue by florianletsch** [cvondrick/vatic#46]
**Expected:** When I mark an annotation as "out of frame" I would expect it to stay selected until a certain frame, when I disable it again. In between I would expect vatic to generate frames with the "of of frame" attribute set to 1.

**What actually happens:** Apparently I would need to set the attribute manually for every single frame. This is quite tedious and not usable for a large amount of frames.

Is this expected behaviour or am I missing something?

A video where you can see that only single frames keep this attribute: http://quick.as/QRgxSnmJ0

_PS: As an aside, I am actually using this attribute as an artificial way to mark the beginning and end of a performed gesture in the video. So I draw an "active" box somewhere in the video and want to mark it as "out of frame" whenever there is no gesture performed._

**Comment by ymm8505** [comment]
when you have some frame‘s box attribute is  ‘out of frame’。 save to DB，only get first ‘out of frame’ box   attribute

**Comment by florianletsch** [comment]
Yes, that's what I described. So... it is intentional then, that it is happening this way? Bummer.

**Comment by ymm8505** [comment]
But when you know this rule，you can modify js Auto create other Frame‘s attribute。
I have modified the JS file, automatically generated the value of the property behind the frame.

**Comment by kr-ish** [comment]
You can modify the estimate function in tracks.js to fix this behavior-

comment the if conditionals that use the "outside" paramater

`        if (bounds['right'] == null )//|| bounds['left'].outside)
        {
            return bounds['left'];
        }

/*        if (bounds['right'].outside)
        {
            return bounds['right'];
        }*/`
